activation/activation.go

```diff
@@ -159,6 +161,12 @@ func WithPostStates(ps PostStates) BuilderOption {
 	}
 }
 
+func BuilderAtxVersions(v AtxVersions) BuilderOption {
+	return func(h *Builder) {
+		h.versions = append(h.versions, v.asSlice()...)
+	}
+}
```

Comment:
I think this option is a bit confusing. For me it should either take 1 version and be called multiple times or take all versions and be called once, i.e.

```suggestion
func BuilderAtxVersions(v AtxVersions) BuilderOption {
	return func(h *Builder) {
		h.versions = v.asSlice()
	}
}
```

Reply:
It appends to `h.versions` because it contains the default value for `epoch 0 -> Version1`. I could change it to make it more clear it's not intended to be called multiple times:
```suggestion
func BuilderAtxVersions(v AtxVersions) BuilderOption {
	return func(h *Builder) {
		h.versions = append([]atxVersion{{0, types.AtxV1}}, v.asSlice()...)
	}
}
```

Reply:
I would prefer the suggested change 🙂